cli/requirements.txt

```diff
@@ -9,6 +9,7 @@ jinja2>=2.11.0
 marshmallow
 aws-cdk.core
 aws-cdk.aws-ec2
+aws-cdk.aws-fsx
```

Comment:
we could add dependency to `cli/test/requirements.txt` as well

Reply:
Let's see what others think about this :+1: